components/SpeechSynthesisModule/SpeechSynthVoiceSelector.js

```diff
@@ -0,0 +1,23 @@
+import StyledButton from "../Buttons/StyledButton";
+
+export default function SpeechSynthVoiceSelector({
+  availableVoices,
+  onSubmit,
+}) {
+  return (
+    <form onSubmit={onSubmit}>
+      <select name="voiceURI">
+        {availableVoices.map((voice) => {
+          return (
+            <option key={voice.name} value={voice.name}>
+              {voice.name} {voice.lang}
+            </option>
+          );
+        })}
+      </select>
+      <StyledButton>
+        <p>set language</p>
```

Comment:
Not sure but maybe its better to say "set voice"?

```suggestion
        <p>set voice</p>
```

Reply:
You're right, I already changed most occurrences, forgot here, thanks!